src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -373,8 +372,7 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 				let kind: TerminalCompletionItemKind | undefined;
 				if (foldersRequested && stat.isDirectory) {
 					kind = TerminalCompletionItemKind.Folder;
-				}
-				if (filesRequested && !stat.isDirectory && (stat.isFile || stat.resource.scheme === Schemas.file)) {
+				} else if (filesRequested) {
```

Comment:
`!foldersRequests && stat.isDirectory` could enter this if you remove `stat.isFile`